app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :users
+  belongs_to :posts
```

Comment:
- Good job done so far, but there are some changes needed to be made, A comment should belong to a `user` not `users`. Likewise, `post` not `posts`. Please fix this 🙏